blogposts/2021/release-3.32.md

```diff
@@ -0,0 +1,53 @@
+changelogItems:
+  - description: The search sidebar now shows a revisions section when all search results are from a single repository. This makes it easier to search in and switch between different revisions.
+    url: https://github.com/sourcegraph/sourcegraph/pull/23835
+    category: Search
+  - description: The various alerts overview panels in Grafana can now be clicked on to go directly to the relevant panels and dashboards.
+    url: https://github.com/sourcegraph/sourcegraph/pull/24920
+    category: Admin
+  - description: '`allowGroupsPermissionsSync` in the GitHub authorization provider is now required to enable the experimental GitHub teams and organizations permissions caching. Permissions caching can significantly reduce the amount of time it takes to perform a full permissions sync.'
+    url: https://docs.sourcegraph.com/admin/repo/permissions#teams-and-organizations-permissions-caching
+    category: Admin
+  - description: Fixed a number of issues where repository permissions sync may fail for instances with very large numbers of repositories.
+    url: https://github.com/sourcegraph/sourcegraph/pull/24972
+    category: Repositories
+  - description: 'Fixed excessive re-rendering of the web application on every keypress in the search query input.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/24844
+    category: Search
+  - description: 'The `PRECISE_CODE_INTEL_DATA_TTL` environment variable is no longer read by the worker service. Instead, site admins can configure global and repository-specific data retention policies in the UI to control the length of time LSIF uploads are considered _fresh_.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/24793
+    category: Admin
+---
+
+Sourcegraph 3.32 is now available! Here are some highlights from this release:
+
+## Code Insights is available in Public Beta
+
+Code Insights is a new analytics tool that lets you track and understand what’s in your code and how it changes over time. You can find it via “Insights” in the main navigation bar. It is available for all Sourcegraph users for free while in beta. When Code Insights is officially released, we may disable your use of it or charge for continued use. We're still polishing Code Insights, so we appreciate it if you [share any bugs or feedback about your experience](mailto:feedback@sourcegraph.com).
+
+## Revisions sidebar
+
+In the left sidebar of your search results, you can now see both branches and tags associated with a repository you’re viewing. When you add a search literal, such as “auth,” the search term will persist across searches, allowing you to search quickly between branches or tags for that search term.
+
+<div style="text-align:center"><video autoplay loop muted playsinline style="width:625px">
+  <source src="https://storage.googleapis.com/sourcegraph-assets/blog/3.32/search_revisions_sidebar.mp4" type="video/mp4">
+</video></div>
```

Comment:
Something to consider for future posts: I like to include a short caption for any image/media embeds so we can draw people in who might otherwise be skimming the post.